sip-core/src/main/java/de/ikor/sip/foundation/core/actuator/routes/MarkdownObject.java

```diff
@@ -0,0 +1,16 @@
+package de.ikor.sip.foundation.core.actuator.routes;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@AllArgsConstructor
+public class MarkdownObject {
+
+  private String fileName;
+  private String fileDescription;
+  private static final String CONTENT = "content";
```

Comment:
Please move declaration of static field before declaration of instance fields, to keep Java convention.